tests/core/modules/test_element_references.py

```diff
@@ -0,0 +1,112 @@
+import pytest
+import numpy.testing as npt
+import numpy as np
+import torch
+
+from fairchem.core.datasets import data_list_collater
+from fairchem.core.modules.element_references import (
+    fit_linear_references,
+    create_element_references,
+    LinearReference,
+)
+
+
+@pytest.fixture(scope="session")
+def element_refs(dataset, max_num_elements):
+    return fit_linear_references(
+        ["energy"],
+        dataset,
+        batch_size=16,
+        shuffle=False,
```

Comment:
For reproducibility would be nice to have a test where we fit two linear references back to back (with same seed) and make sure they are identical. And two different seeds are different?